org.eclipse.dawnsci.plotting.api/src/org/eclipse/dawnsci/plotting/api/EventTrackerServiceLoader.java

```diff
@@ -29,7 +29,7 @@ public EventTrackerServiceLoader() {
 	 * Injected by OSGI
 	 * @param et
 	 */
-	public static void setService(EventTracker et) {
+	public void setService(EventTracker et) {
 		service = et;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)